foundry-voting/README.md

```diff
@@ -1,54 +1,113 @@
-# zk Voting with Foundry
+# Foundry Voting Example
 
-[![Run Tests on PR](https://github.com/noir-lang/noir-starter/actions/workflows/foundry-voting.yml/badge.svg)](https://github.com/noir-lang/noir-starter/actions/workflows/foundry-voting.yml)
+This project demonstrates a voting system implementation using Noir circuits and Foundry.
 
-This example project shows how to create a simple zk voting circuit in Noir with a corresponding Solidity contract to track eligible voters, proposals and votes.
+## Quick Start
 
-This example was last tested with Noir version 0.28.0. You can install it with [noirup](https://noir-lang.org/docs/getting_started/installation/#installing-noirup) using
+Install dependencies:
+```bash
+bun install
+```
+
+## Development Guide
 
+### Interactive Build Process
+
+Run all steps interactively with status updates:
 ```bash
-noirup -v 0.28.0
+bun run build:ultraplonk
 ```
 
-## Overview
+This will guide you through all the steps with prompts and status updates.
+
+### Manual Steps
 
-This is the model used for creating the [circuit](circuits/src/main.nr) and the [zkVote contract](src/zkVote.sol) to manage private voting.
+#### 1. Circuit Testing
 
-1. Create a set of voters. A merkle root is stored in the zkVote Solidity contract that voters will use to verify membership against. In this example, there are 4 accounts in the set of voters. The private keys are 0, 1, 2, 3 and the secret value to create the commitment is 9.
```

Comment:
I feel like we should keep an explanation of how the circuit + contracts work, so readers can get a sense of what is going on without having to read all of the code. wdyt?

Reply:
yeah make sense. I've put it back.